src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/separatorwrap/Example3.java

```diff
@@ -0,0 +1,26 @@
+/*
+SeparatorWrap
+tokens = COMMA
+option = nl
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.separatorwrap;
+
+class Example3 {
+  String s;
+
+  int a,
+    b; // violation above '',' should be on a new line'
+
+  void foo(int a,
+                int b) { // violation above '',' should be on a new line'
+    int r
+      , t; // OK
```

Comment:
Please do not copy `ok` comments.

Reply:
I forgot to say. These comments I felt should be left to say. Don't know why. Please let me know if I really should remove them.

Reply:
So there should be a reason defined in such comments.
Just "ok" is not ok

Reply:
Done. Added explanation.